src/server/dfly_main.cc

```diff
@@ -161,35 +161,39 @@ struct VersionMonitor {
   }
 
  private:
-  void RunTask(SSL_CTX* ssl_ctx);
+  struct ssl_deleter {
+    void operator()(SSL_CTX* ssl) {
+      if (ssl) {
+        free(ssl);
```

Comment:
why not TlsClient::FreeContext?

Reply:
why do you call free here?

Reply:
Because I am an idiot 🤣 Thank you for spotting this Adi ❤️